src/libraries/System.Net.Ping/tests/FunctionalTests/System.Net.Ping.Functional.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-OSX;$(NetCoreAppCurrent)-iOS;$(NetCoreAppCurrent)-tvOS;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
```

Comment:
To remove the code that runs `ping` command line utility. Alternatively it could be done in a way that the code can get linked out but from the comments on earlier PR this looked like a preferred alternative.

Reply:
https://github.com/dotnet/runtime/pull/42006#discussion_r485373089

Reply:
But this is a test assembly not implementation which needs to be platform specific

Reply:
Ah, right. If the test assembly was built with just `-Unix` target it didn't pick up the implementation from `-OSX`/`-iOS` implementation assemblies. It could have been a quirk of some local build though. I'll try to revert it locally and do a full build to see if that still happens.

Reply:
can you please check again whether this is really needed? I don't see why it would be.

Reply:
It's not. I was pretty sure I reverted it but apparently GIT thinks differently :-/

Reply:
For the record, the reason why I hit it in the first place is that I usually rebuild single projects using `../../../dotnet.sh build src` and then run tests by `arch -arch x86_64 ../../../dotnet.sh build tests /t:Test /p:TargetOS=iOSSimulator /p:DevTeamProvisioning=xxx /p:Outerloop=true`. This works most of the time by it fails when new targets are added in the library project. The new targets do get built but the test project doesn't consume them. I need to do full rebuild from the top with `./build.sh mono+libs+libs.pretest` (or similar) to pick that change up.